### DISCUSSION TITLE
屏蔽首页右下角的“稍后再看”按钮

### DISCUSSION THREAD

**Discussion by MJSummer** [the1812/Bilibili-Evolved#4936] [功能建议 / Ideas]
如题。点击会提示“浏览器暂不支持此功能”，尽管内核版本是122，但是屏蔽它一劳永逸，还是右上角的入口更为方便。

**Comment by MJSummer** [comment]
![image](https://github.com/user-attachments/assets/66feb2d4-fec8-4a12-bf25-7b2428a877ca)
点进去是个队列浮窗，没法全屏，这也太鸡肋了

**Reply by Geeyun-JY3** [reply]
没搞错的话这功能是抄的 YouTube 的待播列表，不过破站这个好像还更进了一步，做成了画中画。
